### DISCUSSION TITLE
Requested review from empty?

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#16157] [Pull Requests]
https://github.com/nginxinc/kubernetes-ingress/pull/2641#event-6564840784

<img width="799" alt="image" src="https://user-images.githubusercontent.com/2119212/167175742-67b77894-8b01-4f9b-b7c8-7f7d689952df.png">

> @[lucacome](https://github.com/lucacome) lucacome requested review from , [haywoodsh](https://github.com/haywoodsh) (assigned from nginxinc/kic) and [jjngx](https://github.com/jjngx) (assigned from nginxinc/kic) [6 minutes ago](https://github.com/nginxinc/kubernetes-ingress/pull/2641#event-6564840784)

**Comment by vedantmgoyal9** [comment]
Another one:

![image](https://user-images.githubusercontent.com/83997633/167176131-0e250f2e-8e95-4899-bded-6ce47657284e.png)
